### ISSUE TITLE
PC instantly shuts down after pressing YES on the first time setup menu with default settings

### ISSUE THREAD

**Issue by BlenderEngineer** [Rem0o/FanControl.Releases#2568]
**Describe the bug**
PC instantly shuts down after pressing YES on the first time setup menu with default settings

**Is there a log.txt file next to FanControl.exe with recent date entries?**
No

**Relevant hardware specs and setup**
NVIDIA GeForce RTX 2060
Intel i7 7700, stock CPU fan
16GB RAM
Cooler Master case with 3 ball-bearing fans in front

**Comment by Rem0o** [comment]
This?

![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/18c9991a-d2ee-4747-8a08-38c0ff636829)

**Comment by BlenderEngineer** [comment]
Exactly

**Comment by BlenderEngineer** [comment]
Doesn't even go through blue screen. PC turns off completely.

**Comment by Rem0o** [comment]
That's very odd.  Pressing yes doesn't do anything except opening the next dialog menu.  There is nothing but UI at that point. 

If you refuse the initial setup and go here:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/2c1d2ce6-5970-4057-8ec5-9d120b727992)

Can you get to that menu?  That's the equivalent of pressing "yes" where you were.

**Comment by BlenderEngineer** [comment]
Yes, I can refuse and get to that menu. Pressing Ok on the menu in the screenshot, however causes the same thing.

**Comment by BlenderEngineer** [comment]
Interesting... Going through that sensors menu seems to work fine.

**Comment by BlenderEngineer** [comment]
Pressing this button also instantly shuts down the PC

![image](https://github.com/Rem0o/FanControl.Releases/assets/82451249/ea9c3f0a-32ac-41c0-bff8-972ddcedf540)

**Comment by BlenderEngineer** [comment]
Manual pairing button works fine, however.
